documentation/docs/fragment.md

```diff
@@ -59,6 +59,34 @@ is no special field for the fragment necessary. So if this is the query:
 
 Then `user` will be of the type `Avatar_User.t`.
 
+#### Referencing fragments from other modules
```

Comment:
Not sure about the placement of this section relative to the other ones, but it felt more important to me than variables in fragments or `ppxAs` so I put it ahead of them.